docs/en/writing-running-appium/android/android-appbundle.md

```diff
@@ -0,0 +1,51 @@
+## How To Test Android App Bundle
+
+Google has released [Android App Bundle](https://developer.android.com/platform/technology/app-bundle/) feature.
+`.aab` file is generated by the feature. We must upload the file to Google Play Store.
```

Comment:
An `.aab` file is generated by the feature, which we are supposed to upload to the Google Play Store